HACKING.md

```diff
@@ -80,3 +80,18 @@ source .tox/unit/bin/activate
 pytest -v -k RealPebble
 ```
 
+# Publishing a Release
+
+To make a release of the Operator Framework, do the following:
+
+1. Visit the [releases page on github](https://github.com/canonical/operator/releases).
+2. Click "Draft a new release"
+3. The "Release Title" is simply the full version number, in the form <major>.<minor>.<patch>
+   E.g. 2.3.12
+4. Drop notes and a changelog in the description.
+5. When you are ready, click "Publish". (If you are not ready, click "Save as Draft".)
+
+This will build a Python package and publish it to PyPI.
+
+See .github/workflows.publish.yml for internals. You can troubleshoot errors on the [Actions Tab](https://github.com/canonical/operator/actions)
```

Comment:
```suggestion
This will trigger an automatic build for the Python package and publish it to PyPI (the
API token/secret is already set up in the repository settings).

See [.github/workflows/publish.yml](https://github.com/canonical/operator/blob/main/.github/workflows/publish.yml) for details.
You can troubleshoot errors on the [Actions Tab](https://github.com/canonical/operator/actions).
Don't forget to update the major version number in `publish.yml` if/when that ever changes.
```

Reply:
You're right.  Feel free to adjust accordingly

Reply:
Added your suggestions, minus the last line :-)

(You're not the first one to try to change those version numbers in publish.yml, though -- I added a note about that, too.)